include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -195,6 +195,24 @@ def TTNN_MaximumOp :  TTNN_ElementwiseBinaryOp<"maximum"> {
     }];
 }
 
+class TTNN_BroadcastOp<string mnemonic, list<Trait> traits = []> : TTNN_NamedDPSOp<mnemonic, traits> {
+    let summary = "Broadcast op.";
+    let description = [{
+      Reduction op.
+    }];
+
+    let arguments = (ins AnyRankedTensor:$input,
+                         AnyRankedTensor:$output,
+                         I64ArrayAttr:$dimension,
+                         TT_OperandConstraintArrayAttr:$operand_constraints);
+
+    let results = (outs AnyRankedTensor:$result);
+
+    let extraClassDeclaration = [{
+      MutableOperandRange getDpsInitsMutable() { return getOutputMutable(); }
+    }];
+}
+
```

Comment:
TTNN doesn't have a broadcast op and we have the pass that bypasses it below, so we don't need this tablegen def.

Reply:
Ok I have removed it.